python/lsst/afw/multiband.py

```diff
@@ -82,6 +83,15 @@ def clone(self, deep=True):
         """
         pass
 
+    @property
+    def filters(self):
+        """List of filter names for the single band objects (deprecated)
+
+        Use `bands` instead.
+        """
+        warnings.warn("Use `bands` instead of `filters`", DeprecationWarning)
```

Comment:
You could use the same presciption to deprecate the `staticmethod` so it appears in the documentation as well.
https://developer.lsst.io/stack/deprecating-interfaces.html#python-deprecation

Reply:
I initially tried that but unfortunately it doesn't work for properties.

Reply:
Hmm, I tried it locally before commenting this. The order matters - property on top and deprecated below that.

Reply:
Oh... it's because I forgot to import deprecated. :(